gearmand/gearmand.cc

```diff
@@ -264,12 +264,14 @@ int main(int argc, char *argv[])
         std::vector<std::string> args;
         std::copy(tok.begin(), tok.end(), back_inserter(args));
 
-        for (std::vector<std::string>::iterator iter= args.begin();
-             iter != args.end();
-             ++iter)
-        {
-          std::cerr << *iter << std::endl;
-        }
+      	#if defined(DEBUG) && DEBUG
```

Comment:
all other `#if` directives are placed at the beginning of of line.
`git grep '#if' -- '*.cc'`
Could you replace `#if` and `#endif` at the line beginning, please.